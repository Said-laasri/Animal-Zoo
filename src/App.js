import React from 'react';
import { Routes, Route } from 'react-router-dom';
import Header from './navigation/Header';
import Deaths from './component/Deaths';
import './App.css';

function App() {
  return (
    <div className="App">
      <Header />
      <Routes>
        <Route path="/" element={<Deaths />} />
      </Routes>
    </div>
  );
}

export default App;
