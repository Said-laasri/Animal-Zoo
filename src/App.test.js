import Renderer from 'react-test-renderer';
import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from './redux/configStore';
import App from './App';

// test app
it('renders correctly', () => {
  const tree = Renderer.create(
    <React.StrictMode>
      <Provider store={store}>
        <BrowserRouter>
          <App />
        </BrowserRouter>
      </Provider>
    </React.StrictMode>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
