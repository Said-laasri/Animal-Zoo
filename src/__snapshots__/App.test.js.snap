// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="header"
  >
    <h1>
      Animal ZOO
    </h1>
  </div>
  <div
    className="animalList"
  />
</div>
`;
